recipe/meta.yaml

```
@@ -66,11 +68,13 @@ outputs:
         - cmake
         - make
         - llvm-openmp  # [osx]
-        - libgomp      # [linux]
+        - libgomp      # [linux or win]
       host:
         - cuda-version {{ cuda_compiler_version }}  # [cuda_compiler != "None"]
         - nccl                                      # [linux and cuda_compiler != "None"]
+        - libgomp                                   # [win]
       run:
+        - libgomp                                   # [win]
         - __cuda                                    # [cuda_compiler != "None"]
         - cuda-version >=11.2,<12.0a0               # [(cuda_compiler_version or "").startswith("11")]
         - cuda-version >=12.0,<13.0a0               # [(cuda_compiler_version or "").startswith("12")]
```

Swapping the order to match the sections above

```suggestion
      run:
        - __cuda                                    # [cuda_compiler != "None"]
        - cuda-version >=11.2,<12.0a0               # [(cuda_compiler_version or "").startswith("11")]
        - cuda-version >=12.0,<13.0a0               # [(cuda_compiler_version or "").startswith("12")]
        - libgomp                                   # [win]
```